### PR TITLE
Add process for requesting a DNS update to the README.

### PR DESCRIPTION
**Pull request by brendandburns**
@spiffxp @dims @thockin

### CODE REVIEW

File: dns/README.md [hunk 1]
```diff
@@ -29,6 +29,37 @@ We host it in Google Cloud DNS.
...
+
+The process for requesting an update uses Github Issues and PRs.
+
+### Update Request Issue
+
+Open a new issue on this repository with the title "DNS Update Request"
+
+In the issue, please list the following details:
+   * If this update is a create, delete or update.
+   * The complete data for the existing DNS record if applicable (Updates, Deletes).
```
Comment (dims) [inline dns/README.md]: Could we give an example please?

Comment (brendandburns) [reply dns/README.md]: done, please take another look.

File: dns/README.md [hunk 2]
```diff
@@ -29,6 +29,66 @@ We host it in Google Cloud DNS.
...
+In the issue, please list the following details:
+   * If this update is a create, delete or update.
+   * The base domain that is being modified (e.g. "k8s.io")
+   * The complete data for the existing DNS record if applicable (Updates, Deletes).
+   * The complete data for the new DNS record if applicable (Creates, Updates).
+   * The reason for the update.
+
+Once this issue is created, it should be acknowledged by a DNS administrator.
+
+### Example update issue content:
```
Comment (spiffxp) [inline dns/README.md]: I think this work well as an issue template. Move this content to .github/ISSUE_TEMPLATE/dns-request.md

Start that file with something like:
```
name: DNS Update Request
about: Request a create, delete or update of a kubernetes.io or k8s.io DNS record
 ---

(then all of your example content / instructions here)
```

This has worked pretty well for us at k/org, eg: https://github.com/kubernetes/org/issues/new?template=membership.md

I think you would also need to add a default / "general" issue template, we went with https://raw.githubusercontent.com/kubernetes/org/master/.github/ISSUE_TEMPLATE/general-support.md but honestly I think you wouldn't need the add'l content just the name:/about:

Comment (brendandburns) [reply dns/README.md]: Added.

**Comment by brendandburns** [comment]
@spiffxp comments addressed please take another look.

**Comment by spiffxp** [comment]
/lgtm
/cc @hh 
since I believe you're working on verifying DNS changes

**Comment by brendandburns** [comment]
Is anything needed in order to merge this?

**Comment by dims** [comment]
@brendandburns there was a note from Devan Carpenter (@dvn0 ?) on Thu (Dec 20th) on slack

```
Hi channel. I've been working with @hh today on automating the DNS zone manipulation using octodns. It's looking good. We should have more updates tomorrow.
```

**Comment by ii-dev** [comment]
> @brendandburns there was a note from Devan Carpenter (@dvn0 ?) on Thu (Dec 20th) on slack

That's actually me! That message was a bit too optimistic considering how close we are to the holidays, and as some of us are actually in timezones a day ahead of the US. 

My understanding is that the last task before merging this is to add an intermediate staging & verification step to the update process. That's what we are working on.

**Comment by spiffxp** [comment]
ping @hh @devaii @thockin 
I'd like to share this with the community, docs sure would help, is this ready to merge?

File: .github/ISSUE_TEMPLATE/dns-request.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: DNS Update Request
+about: Request a create, delete or update of a kubernetes.io or k8s.io DNS record
+ ---
+ 
+ *Type of update:* Update
```
Comment (thockin) [inline .github/ISSUE_TEMPLATE/dns-request.md]: spurious whitespace at start of line?

**Comment by spiffxp** [comment]
/lgtm
trying to give tide a poke, not sure why this hasn't been merged yet

**Pull request merged**